composer.json

```diff
@@ -14,8 +14,15 @@
     "maxmind-db/reader": "1.6.0",
     "woocommerce/action-scheduler": "3.0.1",
     "woocommerce/woocommerce-blocks": "2.5.11",
-    "woocommerce/woocommerce-rest-api": "1.0.7"
+    "woocommerce/woocommerce-rest-api": "1.0.7",
+    "woocommerce/woocommerce-admin": "0.25.0"
   },
+  "repositories": [
```

Comment:
We can remove this now, since WooCommerce Admin is published in https://packagist.org/packages/woocommerce/woocommerce-admin